packages/core/src/shared/telemetry/vscodeTelemetry.json

```diff
@@ -1131,6 +1131,17 @@
             ],
             "passive": true
         },
+        {
+            "name": "auth_signInPageOpened",
+            "description": "When the Amazon Q sign in page is opened and focused.",
+            "metadata": [
+                {
+                    "type": "source",
```

Comment:
why not add this to the existing metric? https://github.com/aws/aws-toolkit-common/blob/ccb16ca4f73b07c4e1cd79da159312c9ffe403a7/telemetry/definitions/commonDefinitions.json#L2915

Reply:
I'm being rushed to get the change out for this release, but I will port this over after